notebooks/community/feature_store/sdk-feature-store.ipynb

```diff
@@ -0,0 +1,1255 @@
+{
```

Comment:
<p>update the link</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
fixed